ui/i18n/locales/hu_HU/opensnitch-hu_HU.ts

```diff
@@ -6,656 +6,656 @@
     </message>
     <message>
         <location filename="../../../opensnitch/res/preferences.ui" line="441"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The default duration will take place when there&apos;s no UI connected.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Az alapértelmezett időtartam akkor kerül végrehajtásra, ha nincs csatlakoztatva felhasználói felület.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location filename="../../../opensnitch/res/preferences.ui" line="444"/>
         <source>Default duration</source>
-        <translation type="unfinished"></translation>
+        <translation>Alapértelmezett időtartam</translation>
     </message>
     <message>
         <location filename="../../../opensnitch/res/preferences.ui" line="457"/>
         <source>Apply configuration to all nodes</source>
-        <translation type="unfinished"></translation>
+        <translation>Beállítások alkalmazása az összes csomópontra</translation>
     </message>
     <message>
         <location filename="../../../opensnitch/res/preferences.ui" line="480"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The default action will take place when there&apos;s no UI connected.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Az alapértelmezett művelet akkor kerül végrehajtásra, ha nincs csatlakoztatva felhasználói felület.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location filename="../../../opensnitch/res/preferences.ui" line="483"/>
         <source>Default action</source>
-        <translation type="unfinished"></translation>
+        <translation>Alapértelmezett művelet</translation>
     </message>
     <message>
         <location filename="../../../opensnitch/res/preferences.ui" line="496"/>
         <source>HostName</source>
-        <translation type="unfinished"></translation>
+        <translation>Állomásnév</translation>
     </message>
     <message>
         <location filename="../../../opensnitch/res/preferences.ui" line="526"/>
         <source>proc</source>
-        <translation type="unfinished"></translation>
+        <translation>proc</translation>
     </message>
     <message>
         <location filename="../../../opensnitch/res/preferences.ui" line="531"/>
         <source>ebpf</source>
-        <translation type="unfinished"></translation>
+        <translation>ebpf</translation>
     </message>
     <message>
         <location filename="../../../opensnitch/res/preferences.ui" line="536"/>
         <source>audit</source>
-        <translation type="unfinished"></translation>
+        <translation>naplózás</translation>
```

Comment:
don't translate this string, it's the name of the process monitor method (auditd).

Sorry for that, I'll make it not translatable for future releases.

Reply:
done